### PR TITLE
Secondary colors

### PR DESCRIPTION
**Pull request by mansona**
Add the secondary colour pallet as defined in the Abstract

https://deploy-preview-305--ember-styleguide.netlify.app/concepts/colours/

localhost_4200_concepts_colours

### CODE REVIEW

File: docs/concepts/colours.md [hunk 1]
```diff
@@ -29,14 +29,23 @@ The primary palette is applied across every page of the website and contains the
 
 The secondary palette is applied to UI elements and it's not part of the base colors. The purpose of the secondary palette is to ensure the readability, usability, and accessibility of all UI elements and enhance the communication of actions, changes in state, or errors.
```
Comment (MelSumner) [inline docs/concepts/colours.md:30]:
```suggestion
These colors are for charts and diagrams only. The secondary palette is applied to UI elements and is not part of the base colors. The purpose of the secondary palette is to ensure the readability, usability, and accessibility of charts and diagrams and enhance the communication of actions, changes in state, or errors.
```

**Comment by mansona** [comment]
@MelSumner I have applied your suggestion and rebased this branch 👍 

What is left now to be able to get this merged? I seem to remember that we discussed merging this and iterating rather than blocking it 🤔 but I can't remember the planned way forward

**Comment by MelSumner** [comment]
@mansona we absolutely cannot merge something that touches branding w/o input from @wifelette . It's fine to be patient with this one for now. 👍

**Comment by mansona** [comment]
@MelSumner the reason I'm asking is because https://github.com/ember-learn/ember-styleguide/pull/306 is currently built on top of this, using one of the new secondary colours. I think #306 is the last blocker for the release of ember-styleguide and also https://github.com/ember-learn/ember-blog/pull/598

I'm happy to be patient on this particular change, but in the interest of not blocking the release I was wondering if you thought it would be ok for me to alter #306 to just use one of the existing colours instead of being blocked on this? thoughts?

**Comment by MelSumner** [comment]
I'm not sure why it's a blocker tbh, the current design should be fine, it fits in okay (even if not super perfectly) and also it's used on the guides and api docs which we haven't gotten to yet. So the logic here just doesn't connect for me.

**Comment by mansona** [comment]
@MelSumner as it's unrelated to Secondary Colours now I will continue this discussion on #306 👍

**Comment by wifelette** [comment]
Sorry y'all, missed the ping on this one! I'm consulting with @jeffdaley to see if he has thoughts (since we'd want to incorporate these into the official branding guide as well). Pulling colors from the random illustrations is mildly...random, but if they happen to work out well, they're just as good as any color choices :p 

Jeff won't be around until Monday but hopefully he/we will have thoughts Monday 👍

**Comment by jeffdaley** [comment]
Looks good, though the text is not 100% clear to me.

> These colors are for charts and diagrams only. The secondary palette is applied to UI elements...

If the palette can be used on UI elements, could we give some examples? Otherwise if it's just for charts and diagrams, maybe we call the palette "Illustration Colors" to more clearly narrow its use.

Styleguide looks great, btw!

**Comment by amyrlam** [comment]
I'm not sure what we mean by UI elements here, outside of charts and diagrams. @MelSumner re-requested for review, can you take another look? I can also help get this merged, am going through the PR list locks posted as have some cycles.

**Comment by MelSumner** [comment]
Calling it "Illustration Colors" sounds good to me. 👍 

Thanks for following up @amyrlam

**Comment by amyrlam** [comment]
I'm not sure how to fix the build failure yet TBH.

**Comment by ijlee2** [comment]
@amyrlam It looks like the `{{on}}` modifier in `<EsHeader>` component is throwing an error in `release` (3.19), `beta`, and `canary`. I think the error was addressed in the following issues:

- https://github.com/ember-learn/ember-styleguide/pull/312
- https://github.com/ember-polyfills/ember-on-modifier/pull/289/files
- https://github.com/emberjs/ember-render-modifiers/pull/13/files

I think we can try 2 things.

1. Rebase this branch to make sure that we are using the latest `@ember/render-modifiers` and `ember-on-modifier` defined in `package.json`.
2. Make a separate PR to uninstall `ember-on-modifier` (i.e. use native `{{on}}` modifiers instead of polyfills) now that this addon uses `ember-source@3.17`. After merging that PR, rebase this branch.

**Comment by amyrlam** [comment]
@ijlee2 Thanks, your comment helped me realize that this PR was targeting a non-master branch that was already merged. Changed it to master and merging in!

**Pull request merged**